alembic/versions/8c6cddf6335d_add_panels_and_bands_plugins.py

```diff
@@ -0,0 +1,180 @@
+"""Add panels and bands plugins.
+
+Revision ID: 8c6cddf6335d
+Revises: 
+Create Date: 2016-05-11 15:26:23.703464
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '8c6cddf6335d'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+import sideboard
```

Comment:
This three import lines needed to be added manually, but I would be surprised if we couldn't do that automatically.